packages/gatsby-cli/src/structured-errors/error-map.js

```diff
@@ -89,6 +89,77 @@ const errorMap = {
     type: `PLUGIN`,
     level: `ERROR`,
   },
+  "11322": {
+    text: context =>
+      `${
+        context.pluginName
+      } created a page and didn't pass the path to the component.\n\nThe page object passed to createPage:\n${JSON.stringify(
+        context.pageObject,
+        null,
+        4
+      )}\n\nSee the documentation for the "createPage" action — https://www.gatsbyjs.org/docs/actions/#createPage`,
```

Comment:
Should we move to docsUrl instead? Or is docsUrl not really for this usecase
```suggestion
      )}`,
    docsUrl: `https://www.gatsbyjs.org/docs/actions/#createPage`
```

Reply:
This is why we need MDX actually — `docsUrl` is nice but it hardcodes the layout of stuff where I want a link but I want to preface it with a cue for when you'd need to go visit the page.

So I'll leave it as is for now.